KILTprotocol/docs
# fix(deps/ci): add node-fetch@^2 as direct dependency, fix frozen install

## quick fix 

Node-fetch was used in a test file but not added as a direct dependency, resulting in a failure to run tests when it was bumped to v3 by the dependency that required it ([here](https://github.com/KILTprotocol/docs/runs/7391749275?check_suite_focus=true)). Added `node-fetch@^2.6.7` as a direct dev-dependency to resolve.

Additionally, refreshing the lock files revealed that they were out of sync with their respective `package.json`. This went unnoticed because the workflows used the command `yarn install --frozen`, but the `--frozen` option does not exist. Fixed that too (it's `--frozen-lockfile`). 

## How to test:

I can revert the lock file refresh to prove my point I guess (see https://github.com/KILTprotocol/docs/actions/runs/2698400017).
Also checked that the issue with the node-fetch import is now resolved.

## Checklist:
